docs/essentials/widget_language.md

```diff
@@ -387,25 +386,25 @@ finally {
 
 > Tips
 >
-> In most cases, `let_watch!` is a more convenient, but `watch!` is more flexible in some complex situations.
+> In most cases `let_watch!` is more convenient. `watch!` is more flexible in some complex situations.
 
 ### Access `BuildCtx` in `init` block
 
-Our `greet` text with a default style. In this section, we'll use larger letters to highlight it. Instead of hard code for greet `style`, we'll use the `Theme` to init it. 
+So far, our `greet` text was rendered with the default style. In this section we'll use larger letters to emphasize the text. Instead of hard coding `style` for `greet`, we will use a theme to customize it. 
 
-First, let's add an `init` block after `states`
+At first we add an `init` block after the `states` block.
 
 ```rust
 init ctx => {
   let style = TypographyTheme::of(ctx).headline1.text.clone();
 }
 
 ```
-`init` block has the same syntax as `finally`, but the above code has a little more stuff (`ctx =>`) than what the `finally` block we used before.
+The `init` block has the same syntax as the `finally` block, but the above code has some more stuff (`ctx =>`) than the `finally` block we used before.
 
-`ctx =>` is a syntax we use to name `BuildCtx`. Here, `ctx` is the name of `BuildCtx`. Because the `finally` block not use `BuildCtx`, so it be omitted. Let deep into the body, we create a style variable, and init it by the predefined theme.
+`ctx =>` is syntax to name `BuildCtx`. Here, `ctx` is the name assigned to `BuildCtx`. Because the `finally` block does not use `BuildCtx` it is omitted. Lets now look deep into the body. We created a style variable, and initialized it with the predefined theme.
 
-Second, we use `style` to init the `greet` text.
+The we use the `style` field to assign the themes style to the `greet` text.
```

Comment:
First " The" should be " Second" or "Secondary"?

Reply:
I had a bit an issue with "First" and "Second" being so far away from each other, so that it's not easy to spot what "Second" refers to. So I thought to just drop it. It's read from top to bottom anyway and so the text above comes first, the the text further down comes after it. Looks I got list in the midst of it.